configs/sites/cheyenne/modules.yaml

```diff
@@ -2,3 +2,6 @@ modules:
   default:
     enable::
     - lmod
+    lmod:
+      blacklist:
+      - ecflow
```

Comment:
What's the reason for blacklisting ecflow?

Reply:
We are already creating ecflow modules outside of spack-stack (which makes sense, because the manually built ecflow can be used independently of spack-stack stuff). Having another ecflow module in spack-stack then gives these confusing messages "reloading module ...". Also, the auto-generated module file doesn't set the `PYTHONPATH`. This is something that should be fixed in the spack package (you may need to check that on hera - I believe you build ecflow with spack-stack).